src/resultsdisassemblypage.cpp

```diff
@@ -112,19 +137,57 @@ void ResultsDisassemblyPage::showDisassembly()
     // Show empty tab when selected symbol is not valid
     if (m_curSymbol.symbol.isEmpty()) {
         clear();
-        return;
     }
 
     // Call objdump with arguments: addresses range and binary file
     QString processName = m_objdump;
     QStringList arguments;
     arguments << QLatin1String("-d") << QLatin1String("--start-address") << QLatin1String("0x") + QString::number(m_curSymbol.relAddr, 16) <<
             QLatin1String("--stop-address") << QLatin1String("0x") + QString::number(m_curSymbol.relAddr + m_curSymbol.size, 16) <<
-            m_curSymbol.path;
+            m_curAppPath;
 
     showDisassembly(processName, arguments);
 }
 
+QByteArray ResultsDisassemblyPage::processDisassemblyGenRun(QString processName, QStringList arguments)
```

Comment:
you are again doing multiple things in one commit - but as I said, remove the wheel zoom feature and then only keep the diagnostics here

Reply:
also: const& the function args